tests/clients/redis/test_redis_client.py

```diff
@@ -25,6 +26,105 @@ def delete_mock():
     return Mock(return_value=4)
 
 
+@pytest.fixture(scope="function")
+def redis_client_with_live_instance(app, tmp_path_factory):
+    # note that this fixture cannot be used in parallel without the risk
+    # of race conditions. Use the decorator @pytest.mark.xdist_group(name="group1")
+    # for tests using this fixture to ensure they run on the same worker.
+    root_tmp_dir = tmp_path_factory.getbasetemp().parent
+    redis_lock_file = root_tmp_dir / "redis_lock"
+    app.config["REDIS_ENABLED"] = True
+    app.config["REDIS_URL"] = "redis://localhost:6999/0"
```

Comment:
These values are only called once, not sure if there is any benefit in refactoring them out into constants or not?